src/execution/window_executor.cpp

```diff
@@ -594,6 +597,159 @@ void WindowExecutorBoundsState::UpdateBounds(idx_t row_idx, DataChunk &input_chu
 	state.Bounds(bounds, row_idx, range, count, boundary_start, boundary_end, partition_mask, order_mask);
 }
 
+//===--------------------------------------------------------------------===//
+// ExclusionFilter
+//===--------------------------------------------------------------------===//
+
+//! Given a window expression, determine if it has an EXCLUDE clause that must be taken into account
+static bool ExclusionIgnored(BoundWindowExpression &wexpr) {
```

Comment:
This function combines two different questions: 

- Does this function support exclusion
- Is exclusion in play

Individual function objects already know whether they support exclusion, so the first use case seems unnecessary?